docs/library/xilinx/util_adxcvr/index.rst

```diff
@@ -0,0 +1,327 @@
+     - Type
+     - Description
+   * - ``up_clk``
+     - ``input``
+     - System clock, running on 100 MHz
+   * - ``up_rstn`` 
+     - ``input``
+     - System reset, the same as AXI memory map slave interface reset
+
+
+PLL reference clock
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Type
+     - Description
+   * - ``qpll_ref_clk_0``
+     - ``input``
+     - Reference clock for the QPLL
+   * - ``cpll_ref_clk_0``
+     - ``input``
+     - Reference clock for the CPLL
+
+
+RX interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Type
+     - Description
+   * - ``rx_*_p`` 
+     - ``input``
+     - Positive differential serial data input
+   * - ``rx_*_n``
+     - ``input``
+     - Negative differential serial data input
+   * - ``rx_out_clk_*``
+     - ``output``
+     - Core logic clock output. Frequency = serial line rate/40
+   * - ``rx_clk_*``
+     - ``input``
+     - Core logic clock loop-back input
+   * - ``rx_charisk_*``
+     - ``output[3:0]``
+     - RX Char is K to the JESD204B IP
+   * - ``rx_disperr_*``
+     - ``output[3:0]``
+     - RX disparity error to the JESD204B IP
+   * - ``rx_notintable_*``
+     - ``output[3:0]``
+     - RX Not In Table to the JESD204B IP
+   * - ``rx_data_*``
+     - ``output[3:0]``
+     - RX data to the JESD204B IP
+   * - ``rx_calign_*``
+     - ``input``
+     - RX enable comma alignment from the JESD204B IP
+
+
+TX interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Type
+     - Description
+   * - ``tx_*_p`` 
+     - ``output``
+     - Positive differential serial output
+   * - ``tx_*_n``
+     - ``output``
+     - Negative differential serial output
+   * - ``tx_out_clk_*``
+     - ``output``
+     - Core logic clock output. Frequency = serial line rate/40
+   * - ``tx_clk_*``
+     - ``input``
+     - Core logic clock loop-back input
+   * - ``tx_charisk_*``
+     - ``input[3:0]``
+     - TX Char is K from the JESD204B IP
+   * - ``tx_data_*``
+     - ``input[31:0]``
+     - TX data from the JESD204B IP
+
+
+Common DRP Interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Type
+     - Description
+   * - ``up_cm_*``
+     - ``IO``
+     - The common DRP interface, must be connected to the equivalent DRP ports
+       of AXI_ADXCVR. This is a QUAD interface, shared by four transceiver
+       lanes. This interface is available only if parameter QPLL_ENABLE is set
+       to 0x1.
+
+
+Channel DRP Interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Pin
+     - Type
+     - Description
+   * - ``up_ch_*``
```

Comment:
Isn't this ``up_rx_*`` and  ``up_tx_*`` ?

Reply:
it is, but I think they decided to put ``up_ch_*`` in the original dokuwiki to make it as generic as possible. I can change it if you think it is more descriptive.